static/js/drafts.js

```diff
@@ -761,6 +762,13 @@ export function initialize() {
     window.addEventListener("beforeunload", () => {
         update_draft();
     });
+    window.addEventListener("storage", (event) => {
+        const ls = localstorage();
+        const drafts = ls.getValueFromStorageEvent(event, KEY);
+        if (drafts !== undefined) {
+            set_count(Object.keys(drafts).length);
+        }
+    });
```

Comment:
Will this end up calling `set_count` when this window is itself the thing doing the save, i.e. have the side effect of restoring the flashing you fixed last week?